src/components/molecules/Notification/Notification.tsx

```diff
@@ -1,36 +1,12 @@
 import {notification} from 'antd';
 
-import {Text} from '@custom-antd';
-
-import Colors from '@styles/Colors';
-
-import {StyledNotificationContainer, StyledNotificationHeader, StyledNotificationMessage} from './Notification.styled';
-
-const statusBGColorMap: {[key: string]: string} = {
-  passed: Colors.lime600,
-  success: Colors.lime600,
-  failed: Colors.pink700,
-  error: Colors.pink700,
-  warning: Colors.yellow400,
-  info: Colors.sky600,
-};
+import NotificationContent from './NotificationContent';
 
 function notificationCall(status: string, title: string, message: string = '', duration: number = 4.5) {
   notification.open({
     message: null,
     placement: 'bottomRight',
-    description: (
-      <StyledNotificationContainer $bg={statusBGColorMap[status]}>
-        <StyledNotificationHeader>
-          <Text className="regular middle">{title}</Text>
-        </StyledNotificationHeader>
-        {message ? (
-          <StyledNotificationMessage>
-            <Text className="regular middle">{message}</Text>
-          </StyledNotificationMessage>
-        ) : null}
-      </StyledNotificationContainer>
-    ),
+    description: <NotificationContent status={status} title={title} message={message} />,
```

Comment:
Why the toast notifications should look differently?

Reply:
It is the same, I am just reusing component